src/localization/languageInitialization.ts

```diff
@@ -0,0 +1,17 @@
+import {localization} from './localization';
+import {Languages} from './languages';
+import en from './translations/en.json';
+import ua from './translations/ua.json';
+import ru from './translations/ru.json';
+
+class LanguageInitialization {
```

Comment:
Initialization this is process not object ...
valid names:
LocalisationManager
Localisation
TranslationManager ... etc

Reply:
Or move this impl to Localisation class taht you aready have